spec/display-buffer-spec.coffee

```diff
@@ -721,6 +733,26 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      #set tabLength higher than 2 to make sure we're not just rounding up; we have to actually clip
```

Comment:
Good idea. I wanted that info in there, but adding it to the spec name is smarter than adding a comment.